cloudevents/formats/avro-turbo-format.md

```diff
@@ -0,0 +1,217 @@
+# Avro Turbo Event Format for CloudEvents - Version 1.0.3-wip
+
+## Abstract
+
+The Avro Turbo Format for CloudEvents defines how events attributes are expressed in
+the [Avro 1.9.0 Specification][avro-spec].
+
+This differs from the [Avro format](avro-format.md) in that:
+
+- It is optimized for performance, preferring a more compact representation.
+- Only supports spec version 1.0 (any changes to spec version, requires 
+  changes to Avro schema, which changes the finger-print, 
+  breaking compatibility).
+- Does not natively support JSON (JSON can be straight-forwardly serialized 
+  to bytes and this was therefore not considered neccessary).
```

Comment:
```suggestion
- It does not natively support JSON (JSON can be straight-forwardly serialized 
  to bytes and this was therefore not considered necessary).
```

Beyond the consistency/typo changes, it's not clear to me what "natively supporting JSON" means here. Perhaps describe what the original Avro format allows? (I assume this is related to `doc`, but I'm not sufficiently versed in Avro to really understand that in detail.)

Reply:
Reading further, it feels like it's more than this - my possibly-hazy understanding is that the original format supports multiple data representations - *including* (but not limited to) JSON maps. (For example, I *think* you could just specify a string.) Whereas the Avro Turbo format requires that data is represented as a sequence of bytes - that's the only representation available. Is that a correct reading?